UNRELEASED.md

```diff
@@ -8,6 +8,8 @@ Use [the changelog guidelines](/documentation/Versioning%20and%20changelog.md) t
 
 ### Enhancements
 
+- Added support to `selectAllTextOnFocus` (boolean prop) to `TextField` ([#5216](https://github.com/Shopify/polaris-react/pull/5216))
```

Comment:
```suggestion
- Added the `selectAllTextOnFocus` prop to `TextField` ([#5216](https://github.com/Shopify/polaris-react/pull/5216))
```

This seems like the general entry for new props but I don't hold any strong opinions

Reply:
THANK YOUU!!!